### PR TITLE
Add rbac rule for cluster resources

### PR DESCRIPTION
**Pull request by plkokanov**
**What this PR does / why we need it**:
Adds rbac rules for cluster resources

**Which issue(s) this PR fixes**:
None

**Special notes for your reviewer**:

**Release note**:

```improvement operator
Added missing rbac rules for cluster resources
```

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,12 +1,12 @@
 ############# builder
-FROM golang:1.13.4 AS builder
+FROM golang:1.14.4 AS builder
```
Comment (ialidzhikov) [inline Dockerfile:2]: Could you please update the version in `go.mod` file?

**Comment by plkokanov** [comment]
Weird, thought I updated it in the previous PR. Nice catch!

**Comment by ialidzhikov** [comment]
@plkokanov , can you run also `make generate`?

**Comment by plkokanov** [comment]
yep rerunning everything

**Comment by plkokanov** [comment]
Pretty sure I ran `make generate` `make verify` and `make check` locally this time. 
Am I missing something else

**Comment by ialidzhikov** [comment]
> Pretty sure I ran `make generate` `make verify` and `make check` `make check-generate` locally this time.
> Am I missing something else

Something like this should do the trick
```
docker run --rm -v $GOPATH/src:/go/src -it golang:1.14 bash

# cd src/github.com/gardener/gardener-extension-os-coreos
# make install-requirements
# make generate
```

**Pull request merged**